### ISSUE TITLE
Get one block (with JSON-data)

### ISSUE THREAD

**Issue by AnTheMaker** [rhondle/BlockChain#3]
@rhondle Hey! Awesome project!! I'm storing JSON as data in the blocks. Now my question is: How can I only get one block (e.g. where "id" in the JSON-data is "123"). Would be
awesome if you help me!!

**Comment by AnTheMaker** [comment]
Oh, I just saw the parameter "magic". Could I use this to store the id??

**Comment by rhondle** [comment]
No, that's not what magic is for. Magic is used to mark the beginning of a block with a known sequence of bytes so that the beginning of a block can be identified from an arbitrary starting point.

I'm not quite clear on what you're asking. If you're trying to look up a specific blockhash, an example of how to do that is in the code. If you're trying to find a block with specific data in it, you will have to implement a separate search/index feature to search your blockchain data.

**Comment by AnTheMaker** [comment]
Okay, thank you very much!
